lib/symboldatabase.cpp

```diff
@@ -1478,6 +1478,14 @@ void SymbolDatabase::createSymbolDatabaseIncompleteVars()
             continue;
         if (tok->varId() != 0)
             continue;
+        if (tok->isCast() && tok->link() && tok->str() == "(") {
```

Comment:
Is it intentional to jump to the ")" here:

    dynamic_cast<A*>(b)

Reply:
Thanks, fixed.